test/lib/usd/plugin/nullApiExporter.cpp

```diff
@@ -61,7 +61,29 @@ REGISTER_EXPORT_JOB_CONTEXT_FCT(Thierry, "Thierry", "Exports for Thierry rendere
 
 REGISTER_EXPORT_JOB_CONTEXT_FCT(SceneGrinder, "Scene Grinder", "Exports to Scene Grinder")
 {
-    return VtDictionary();
```

Comment:
For testing interactively in Maya. Provide a few interesting values to make sure the fields are correctly updated and disabled.